server/testsupport/ciagInductionTestDataBuilder.ts

```diff
@@ -10,7 +10,7 @@ const aLongQuestionSetCiagInduction = (options?: {
   modifiedByDateTime?: string
   workInterestModifiedBy?: string
   workInterestModifiedByDateTime?: string
-}): CiagInduction => {
+}): CiagInductionSummaryResponse => {
```

Comment:
In contrast, I don't understand how this is now a `Summary` response rather than a full induction. Perhaps I need to look at it with a fresh pair of eyes in the morning!